kotlinx-coroutines-core/api/kotlinx-coroutines-core.api

```diff
@@ -321,7 +321,7 @@ public abstract interface annotation class kotlinx/coroutines/DelicateCoroutines
 }
 
 public final class kotlinx/coroutines/DispatchedCoroutine {
-	public static final fun get_decision$FU ()Ljava/util/concurrent/atomic/AtomicIntegerFieldUpdater;
+	public static final synthetic fun get_decision$volatile$FU$kotlinx_coroutines_core ()Ljava/util/concurrent/atomic/AtomicIntegerFieldUpdater;
```

Comment:
According to @SokolovaMaria this is expected API change after update of atomicFU compiler plugin.